### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by faustodc**
@bwilkerson Completed missing sentence in diagnostic-messages.md following #1949 steps.

### CODE REVIEW

File: src/tools/diagnostic-messages.md [hunk 1]
```diff
@@ -5644,7 +5644,7 @@ class C {
 }
 {% endprettify %}
 
-If the instances of the class should be mutable, then remove the
+If the instances of the class should be mutable, then remove the `@immutable` annotation:
```
Comment (bwilkerson) [inline src/tools/diagnostic-messages.md:5647]: Thanks. I dropped a space in the source code from which this was generated which kept a whole line of text from being included in the generated output. The original reviewed text was:

If the instances of the class should be mutable, then remove the annotation, or choose a different superclass if the annotation is inherited:

The original source code will be fixed by https://dart-review.googlesource.com/c/sdk/+/171810. If you could apply the fix manually, that would be much appreciated.

**Review by bwilkerson — APPROVED** [review]

**Comment by bwilkerson** [comment]
@kwalrath This lgtm, but I'm not sure what the process is here and whether you want to be involved.

**Comment by kwalrath** [comment]
I'm happy to wait on the sidelines unless @faustodc needs my help.

File: src/tools/diagnostic-messages.md (hunk 1 above)
Comment (faustodc) [reply src/tools/diagnostic-messages.md]: Great, I updated it with the original source code comment.

**Review by bwilkerson — APPROVED** [review]
Thanks!

**Comment by bwilkerson** [comment]
@kwalrath I don't have the ability to merge the PR.

**Review by kwalrath — APPROVED** [review]
Looks great. Thank you, @faustodc and @bwilkerson!

**Comment by faustodc** [comment]
Thank you for your work @kwalrath  and @bwilkerson 😊

**Pull request merged**